src/analyzer/analyzerbeats.cpp

```diff
@@ -217,13 +217,13 @@ void AnalyzerBeats::storeResults(TrackPointer tio) {
         return;
     }
 
-    BeatsPointer pBeats;
+    mixxx::BeatsPointer pBeats;
     if (m_pPlugin->supportsBeatTracking()) {
         QVector<double> beats = m_pPlugin->getBeats();
         QHash<QString, QString> extraVersionInfo = getExtraVersionInfo(
                 m_pluginId, m_bPreferencesFastAnalysis);
         pBeats = BeatFactory::makePreferredBeats(
-                *tio,
+                tio,
```

Comment:
`tio` is a legacy term from when the Track class was called TrackInfoObject. When you come across it, please rename to `track` or `pTrack`

Reply:
Good. I will do. Already fixed in analyzerbeats.cpp and analyzerbeats.h